src/components/button/button.component.js

```diff
@@ -21,6 +27,18 @@ const renderStyledButton = (buttonProps) => {
     styleProps.href = href;
   }
 
+  let paddingX;
+  switch (size) {
+    case 'small':
+      paddingX = '16px';
```

Comment:
Do we have any rules on when to use a string vs when to use the multiplier?

Reply:
I haven't come up with any, although maybe it should be using the multiplier here, I'll change it. Probably makes sense to use the multiplier where we can